src/entity2input.js

```diff
@@ -1,17 +1,23 @@
-const { camelCase, upperFirst } = require('lodash')
 const { entityField2gql } = require("./helpers/gqlConverters")
+const { defaultConvention } = require("./helpers/stringCase")
 const { checker } = require('suma')
 const { entityValidator } = require('./herbsValidator')
 
-function entity2input(entity) {
+function entity2input(entity, options = {}) {
   const validation = entityValidator(entity)
   if (!checker.isEmpty(validation)) {
     const error = Error('InvalidEntity')
     error.invalidArgs = validation
     throw error
   }
 
-  let gql = `input ${upperFirst(camelCase(entity.name))}Input {\n${entityField2gql(entity)}}`
+  const convention = options?.convention || defaultConvention
```

Comment:
almost there! but there is a problem with `options?.convention` and `options.customName` monopolizing the function for _use case_ naming.
there should be a way to change the convention for _field_ naming and _request_ and _return_ naming (check `requestFieldType2gql`, `usecaseRequest2gql`, `usecaseResponse2gql`, etc. for reference).
It doesn't mean we have to implement it now, but we must have room for these implementations in the future.
So my suggestion is:
`options.convention.inputNameRule` (for the function) for and `options.inputName` (for the custom name) so in the furture it can have `options.convention.fieldNameRule` and `options.convention.returnNameRule`, etc

Reply:
great, I solved it according to the request, it made total sense